README.md

```diff
@@ -1,86 +1,71 @@
-This is a software for developing ontologies and instances of ontologies with
-[NeXus](https://www.nexusformat.org/). The software serves two aims.
+![](https://github.com/nomad-coe/nomad-parser-nexus/actions/workflows/pytest.yml/badge.svg?event=push)
+![](https://github.com/nomad-coe/nomad-parser-nexus/actions/workflows/pylint.yml/badge.svg?event=push)
+![](https://img.shields.io/badge/python-3.8-green.svg)
```

Comment:
I think it would be nice to show all python versions we support (the ones we regularly test with the ci now). I normally get them from pypi and explicitly write each version in pyproject toml (I think in this proj we write only `3`, right?)

Reply:
I've added all three manually for now. We'll need to push to pypi to use something like shields.io.